xds/src/main/java/io/grpc/xds/XdsServerWrapper.java

```diff
@@ -681,10 +690,10 @@ private boolean isPermanentError(Status error) {
             filterInterceptors.add(interceptor);
           }
         } else {
-          call.close(
-              Status.UNAVAILABLE.withDescription("HttpFilterConfig(type URL: "
-                      + filterConfig.typeUrl() + ") is not supported on server-side."),
-              new Metadata());
+          String errorMessage = "HttpFilterConfig(type URL: "
+              + filterConfig.typeUrl() + ") is not supported on server-side.";
+          logger.log(Level.WARNING, errorMessage);
```

Comment:
This one is a bit more complicated because 1) it shouldn't happen, as validation should fail before getting here, and 2) this is only per-RPC now because the interceptors are created per-RPC, but that needs to change anyway.